Implemented or(|) not(!) qualifiers. and(&) is the default.

teonar opened the pull request:
e.g.
!earth !moon
  will display items that doesn't have the words (earth or moon)
|earth |moon
  will display items that have moon or earth
!earth moon
  will display items that does not have earth but does have moon
!earth |moon
  will display items that does not have earth but does have moon

Commit:
Implemented or(|) not(!) qualifiers. and(&) is the default.

e.g.
!earth !moon
  will display items that doesn't have the words (earth or moon)
|earth |moon
  will display items that have moon or earth
!earth moon
  will display items that does not have earth but does have moon
!earth |moon
  will display items that does not have earth but does have moon

hugbug commented on webui/fasttable.js:
```diff
@@ -17,8 +17,8 @@
- * $Revision$
- * $Date$
+ * $Revision: 778 $
+ * $Date: 2013-08-07 16:09:43 -0400 (Wed, 07 Aug 2013) $
```
Please revert these two lines. Then I can merge the pull request without editing.

Looks like you were using the files from svn as the basis for the editing. The files from git doesn't have the revision and date as git doesn't support keywords (unfortunately).

teonar replied:
Apologies, worked from other sources that was in a gz file.

Commit:
Implemented or(|) not(!) qualifiers. and(&) is the default.

e.g.
!earth !moon
will display items that doesn't have the words (earth or moon)
|earth |moon
will display items that have moon or earth
!earth moon
will display items that does not have earth but does have moon
!earth |moon
will display items that does not have earth but does have moon

Commit:
Merge branch 'develop' of https://github.com/teonar/nzbget into develop

teonar committed:
was using svn sources
